src/text-input/src/TextInputField.js

```diff
@@ -48,7 +48,6 @@ const TextInputField = memo(
         validationMessage={validationMessage}
         labelFor={id}
         {...matchedProps}
-        {...restProps}
```

Comment:
Still worth spreading `remainingProps` here?

Reply:
Oh just kidding! See that they're sent below. Nevermind!

Reply:
no that gets spread bellow on the input element